modules/migration-rolling-back-migration-web-console.adoc

```diff
@@ -28,7 +28,7 @@ Rollback is not required if the application was not stopped during migration bec
 .Procedure
 
 . In the {mtc-short} web console, click *Migration plans*.
-. Click the Options menu {kebab} beside a migration plan and select *Rollback*.
+. Click the Options menu {kebab} beside a migration plan and select *Rollback* under *Migration*.
```

Comment:
Comma after "migration plan" and before "and".

Reply:
Not required. Two main clauses do not require a comma.